packages/@uppy/transloadit/src/AssemblyOptions.js

```diff
@@ -62,9 +62,13 @@ class AssemblyOptions {
    * Get Assembly options for a file.
    */
   async #getAssemblyOptions (file) {
-    const options = this.opts
+    if (file == null) return undefined
 
+    const options = this.opts
     const assemblyOptions = await options.getAssemblyOptions(file, options)
+
+    if (file == null) return undefined
```

Comment:
```suggestion
    // We check if the file is present here again, because it could had been removed during the await, e.g. if the user hit cancel while we're waiting for the options
    if (file == null) return undefined

```